packages/protocol/scripts/publish.ts

```diff
@@ -0,0 +1,57 @@
+/* tslint:disable no-console */
+import FormData from 'form-data'
+import fs from 'fs'
+import fetch from 'node-fetch'
+import path from 'path'
+
+const sleep = (ms) => {
+  return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+async function main() {
+  const artifactBasePath = './build/contracts'
+  const artifactPaths = fs.readdirSync(artifactBasePath)
+  const network = process.env.NETWORK || 'development'
+
+  console.log('Uploading sources & metadata')
+  console.log('============================')
+  console.log(artifactPaths)
+
+  for (const _path of artifactPaths) {
+    const artifact = require(path.join(process.cwd(), artifactBasePath, _path))
+
+    console.log()
+    console.log(artifact.contractName)
+    console.log('-'.repeat(artifact.contractName.length))
+
+    const chainId = Object.keys(artifact.networks)[0]
+    const address = artifact.networks[chainId].address
```

Comment:
Hmm, again, we don't have deployment artifacts with deployed addresses available. I think the best course of action would be to use a governance proposal JSON to get contracts and addresses that need verified.

Reply:
When you run the script that is migrating the contracts, this json file is not generating the artifact with a section related with the deployment?
I thought that was the normal truffle migration json generation.

```  "networks": {
    "44787": {
      "events": {},
      "links": {},
      "address": "0x9614944C6cE53EBC10ffc787b4b01cF536fAEf06",
      "transactionHash": "0xb3ca3cb849cbb0e86f2bd5e31aa0788fc1580d36db67437e86001cc11b093271"
    }
  }```